examples/misc/lib/effective_dart/docs_good.dart

```diff
@@ -64,19 +68,17 @@ void miscDeclAnalyzedButNotTested() {
 
   <T>() {
     // #docregion third-person
-    /// Returns `true` if every element satisfies the [predicate].
-    bool all(bool predicate(T element)) => ellipsis();
+    /// Connects to server and fetches results of query.
```

Comment:
YES! We should be doing this everywhere (in my opinion) and especially in this style guide.